### DISCUSSION TITLE
Workflows

### DISCUSSION THREAD

**Discussion by keesvanbemmel** [run-llama/LlamaIndexTS#1141] [General]
Hi!

Any thoughts on when llamaindex workflows will come to LlamaIndexTS?

**Comment by Bruno-Lee** [comment]
I really really appreciate that LlamaIndex workflow can support typescript.

**Comment by leipeng** [comment]
I appreciate that as well

**Comment by marcusschiesser** [comment]
Already released last week, see https://ts.llamaindex.ai/modules/workflows
